tests/test_basics.py

```diff
@@ -37,6 +38,31 @@ def error_processor(event, exc_info):
     assert event["exception"]["values"][0]["value"] == "aha! whatever"
 
 
+def test_auto_enabling_integrations_does_not_break(sentry_init, caplog):
```

Comment:
What's the intention of this test? "Does not break" is too open ended.

Seems like it checks that none of the integrations in the list get enabled.

Reply:
That they don't get enabled, but by break I rather meant that the DidNotEnable and ImportErrors are caught down properly. I'll rename